src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      <p className="Person__age">
+        {age ? `I am ${age}` : ''}
+      </p>
+      <p className="Person__partner">
+        {isMarried
+          ? (`${partnerName} is my ${sex === 'm'
+            ? 'wife'
+            : 'husband'}`)
+          : (`I am not married`)}
```

Comment:
```suggestion
          ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
          :  'I am not married'}
```

You can omit the braces 👍